wordpress-mobile/WordPress-Android
# Updated stories lib hash with static gradle method def to load proper…

Updates the Stories library submodule hash to fix lint issues with parent project

To test:
1. run `./gradlew lintVanillaRelease`
2. verify it builds successfully


PR submission checklist:
